### PR TITLE
Update readmes for formatting

### PR DESCRIPTION
**Pull request by nCastle1**
This PR includes updates to the readmes for each sample to address formatting issues (misrendered titles and unnecessary tags). Additionally, it includes updates to the metadata.json files to make them consistent with the corresponding readmes.

### CODE REVIEW

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Map/AuthorMap/README.md [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-5. Before saving, enter Client ID and Redirect URL OAuth settings (or accept the defaults). These settings are required in order to prompt the user to log in to ArcGIS Online.
-5. Provide info for the new portal item, such as a Title, Description, and Tags. 
-6. Click 'Save Map'. 
-7. After successfully logging in to your ArcGIS Online account, the map will be saved to your default folder.
- 
-You can make additional changes, update the map, and then re-save to store changes in the portal item.
-
-[](Esri Tags: ArcGIS Runtime SDK .NET WPF C# C-Sharp DotNet XAML MVVM)
-[](Esri Language: DotNet)
+1) Pan and zoom to the extent you would like for your map.
```
Comment (mbranscomb) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Map/AuthorMap/README.md]: @nCastle1 We should be consistent in using "." or ")".

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/MapView/ShowCallout/README.md [hunk 2]
```diff
@@ -1,12 +1,8 @@
-# Show Callout
+# Show callout
 
 This sample illustrates how to show callouts on a map in response to user interaction.
 
 <img src="ShowCallout.jpg" width="350"/>
 
-### Instructions
 
-Tap on the map to show that point's coordinates.
```
Comment (mbranscomb) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/MapView/ShowCallout/README.md]: @nCastle1 Were the instructions incorrect?

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Tutorial/AuthorEditSaveMap/README.md [hunk 3]
```diff
@@ -1,18 +1,14 @@
-#Author edit and save a map
+# Author, edit, and save a map
 
-This sample is the solution to the [Author, edit, and save maps to your portal tutorial](https://developers.arcgis.com/net/latest/wpf/guide/author-edit-and-save-maps-to-your-portal.htm). It demonstrates how to author and save a map as an ArcGIS portal item (web map). Saving a map to arcgis.com requires an ArcGIS Online login.
+This samples demonstrates how to author and save a map as an ArcGIS portal item (web map). It is also the solution to the [Author, edit, and save maps to your portal tutorial](https://developers.arcgis.com/net/latest/android/guide/author-edit-and-save-maps-to-your-portal.htm) Saving a map to arcgis.com requires an ArcGIS Online login.
```
Comment (mbranscomb) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Tutorial/AuthorEditSaveMap/README.md]: @nCastle1 Should be "sample" singular?

Comment (mbranscomb) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Tutorial/AuthorEditSaveMap/README.md]: @nCastle1 Is that the correct link - looks like it should be WPF?

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Tutorial/AuthorEditSaveMap/README.md [hunk 4]
```diff
@@ -1,18 +1,14 @@
...
-3. Provide info for the new portal item (Title, Description, and Tags). 
-4. Click 'Save Map to Portal'. 
-5. After successfully logging in to your ArcGIS Online account, the map will be saved to your default folder.
-
-You can make additional changes, update the map, and then re-save to store changes in the portal item.
-
-[](Esri Tags: ArcGIS Runtime SDK .NET WPF C# C-Sharp DotNet XAML MVVM)
-[](Esri Language: DotNet)
\ No newline at end of file
+1) Pan and zoom to the extent you would like for your map.
```
Comment (mbranscomb) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Tutorial/AuthorEditSaveMap/README.md]: @nCastle1 We should be consistent in using "." or ")".

File: src/iOS/Xamarin.iOS/Samples/Map/AuthorMap/ReadMe.md [hunk 5]
```diff
@@ -1,21 +1,15 @@
...
-5. Click 'Save' to open the save dialog.
-6. Provide info for the new portal item, such as a Title, Description, and Tags. 
-7. Click 'Save'. 
-8. After successfully logging in to your ArcGIS Online account, the map will be saved to your default folder. 
- 
-You can make additional changes, update the map, and then re-save to store changes in the portal item.
-
-[](Esri Tags: ArcGIS Runtime SDK .NET Xamarin iOS C# C-Sharp DotNet MVVM)
-[](Esri Language: DotNet)
+1) Pan and zoom to the extent you would like for your map.
```
Comment (mbranscomb) [inline src/iOS/Xamarin.iOS/Samples/Map/AuthorMap/ReadMe.md]: @nCastle1 We should be consistent in using "." or ")".

File: src/iOS/Xamarin.iOS/Samples/MapView/ShowCallout/README.md [hunk 6]
```diff
@@ -1,12 +1,8 @@
-# Show Callout
+# Show callout
 
 This sample illustrates how to show callouts on a map in response to user interaction.
 
 <img src="ShowCallout.jpg" width="350"/>
 
-### Instructions
 
-Tap on the map to show that point's coordinates.
```
Comment (mbranscomb) [inline src/iOS/Xamarin.iOS/Samples/MapView/ShowCallout/README.md]: @nCastle1 Were the instructions incorrect?

File: src/iOS/Xamarin.iOS/Samples/Tutorial/AuthorEditSaveMap/README.md [hunk 7]
```diff
@@ -1,18 +1,14 @@
-#Author edit and save a map
+# Author, edit, and save a map
 
-This sample is the solution to the [Author, edit, and save maps to your portal tutorial](https://developers.arcgis.com/net/latest/ios/guide/author-edit-and-save-maps-to-your-portal.htm). It demonstrates how to author and save a map as an ArcGIS portal item (web map). Saving a map to arcgis.com requires an ArcGIS Online login.
+This samples demonstrates how to author and save a map as an ArcGIS portal item (web map). It is also the solution to the [Author, edit, and save maps to your portal tutorial](https://developers.arcgis.com/net/latest/android/guide/author-edit-and-save-maps-to-your-portal.htm) Saving a map to arcgis.com requires an ArcGIS Online login.
```
Comment (mbranscomb) [inline src/iOS/Xamarin.iOS/Samples/Tutorial/AuthorEditSaveMap/README.md]: @nCastle1 Should be "sample" singular?

Comment (mbranscomb) [inline src/iOS/Xamarin.iOS/Samples/Tutorial/AuthorEditSaveMap/README.md]: @nCastle1 Is that the correct link - looks like it should be iOS?

File: src/iOS/Xamarin.iOS/Samples/Tutorial/AuthorEditSaveMap/README.md [hunk 8]
```diff
@@ -1,18 +1,14 @@
...
-3. Provide info for the new portal item (Title, Description, and Tags). 
-4. Click 'Save Map to Portal'. 
-5. After successfully logging in to your ArcGIS Online account, the map will be saved to your default folder.
-
-You can make additional changes, update the map, and then re-save to store changes in the portal item.
-
-[](Esri Tags: ArcGIS Runtime SDK .NET iOS C# C-Sharp DotNet XAML MVVM)
-[](Esri Language: DotNet)
\ No newline at end of file
+1) Pan and zoom to the extent you would like for your map.
```
Comment (mbranscomb) [inline src/iOS/Xamarin.iOS/Samples/Tutorial/AuthorEditSaveMap/README.md]: @nCastle1 We should be consistent in using "." or ")".

File: src/Android/Xamarin.Android/Samples/Tutorial/AuthorEditSaveMap/README.md [hunk 9]
```diff
@@ -1,18 +1,14 @@
-#Author edit and save a map
+# Author, edit, and save a map
 
-This sample is the solution to the [Author, edit, and save maps to your portal tutorial](https://developers.arcgis.com/net/latest/android/guide/author-edit-and-save-maps-to-your-portal.htm). It demonstrates how to author and save a map as an ArcGIS portal item (web map). Saving a map to arcgis.com requires an ArcGIS Online login.
+This samples demonstrates how to author and save a map as an ArcGIS portal item (web map). It is also the solution to the [Author, edit, and save maps to your portal tutorial](https://developers.arcgis.com/net/latest/android/guide/author-edit-and-save-maps-to-your-portal.htm) Saving a map to arcgis.com requires an ArcGIS Online login.
```
Comment (mbranscomb) [inline src/Android/Xamarin.Android/Samples/Tutorial/AuthorEditSaveMap/README.md]: @nCastle1 Should be "sample" singular?

**Comment by nCastle1** [comment]
@mbranscomb Do you want to give this one last look through or should I go ahead and merge?

**Comment by mbranscomb** [comment]
@nCastle1 Changes look good. There are still multiple cases of the word "samples" when it should be "sample" but those predate your changes. They should be addressed in another issue/PR.

**Comment by nCastle1** [comment]
@mbranscomb I'll make another PR on this branch later once those have been resolved.

**Pull request merged**